python/hyperon/atoms.py

```diff
@@ -92,38 +121,51 @@ class AtomType:
     GROUNDED_SPACE = Atom._from_catom(hp.CAtomType.GROUNDED_SPACE)
 
 class GroundedAtom(Atom):
+    """Grounded Atom represents sub-symbolic knowledge. On the API level it allows
+    keeping data and behaviour inside an atom. There are three aspects of the grounded
+    atom which can be customized:
+
+        - the type of grounded atom is provided by the atom itself;
+        - matching algorithm of the atom can be modified by the user;
+        - atom can be made executable; such atom can be used to apply some sub-symbolic
+          operations to other atoms as arguments.
+    """
 
     def __init__(self, catom):
+        """Initialize a grounded atom"""
         super().__init__(catom)
 
     def get_object(self):
+        """Gets the Grounded Atom object or the space wrapped inside a Grounded Atom"""
         from .base import SpaceRef
         if self.get_grounded_type() == AtomType.GROUNDED_SPACE:
             return SpaceRef._from_cspace(hp.atom_get_space(self.catom))
         else:
             return hp.atom_get_object(self.catom)
 
     def get_grounded_type(self):
+        """Retrieve the grounded type of a Grounded Atom."""
         return Atom._from_catom(hp.atom_get_grounded_type(self.catom))
 
 def G(object, type=AtomType.UNDEFINED):
+    """A convenient method to construct a GroundedAtom"""
     assert hasattr(object, "copy"), "Method copy should be implemented by grounded object"
     return GroundedAtom(hp.atom_gnd(object, type.catom))
 
-"""
-Private glue for Hyperonpy implementation
-"""
 def _priv_call_execute_on_grounded_atom(gnd, typ, args):
+    """
```

Comment:
The idea behind this comment (and others like it) is that it would signal to anybody who built the API docs including the '_' methods that they shouldn't call this method.

Reply:
Exactly. The "_" methods are conventionally treated as private methods.